File: Algoritmos Traduzidos/Python/Algoritmos traduzidos[Luis]/ex5.py
valor=input('Introduza o valor da prestação:')

taxa=input('Introduza a taxa:')

tempo=input('Introduza o numero de meses em atraso:')

prest=float(valor)+(float(valor)*float(taxa)/100)*int(tempo)

print('A prestação em atraso é:', int(prest))

File: Algoritmos Traduzidos/Lua/Algoritmos(LuisSilva)/ex5.lua
io.write'Introduza o valor da prestação:'

valor = io.read()

io.write'Introduza a taxa:'

taxa = io.read()

io.write'Introduza o numero de meses em atraso:'

tempo = io.read()

prest=valor+(valor*(taxa/100)*tempo)

print('A prestação em atraso é:',prest)
